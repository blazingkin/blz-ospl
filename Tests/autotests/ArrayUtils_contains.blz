:main
    #Test 00 :contains
	print(">>> Test 00 :contains <<<")
	arr = [1,2]
	print(arr.contains?(1))
	print("--------------------------------")
end