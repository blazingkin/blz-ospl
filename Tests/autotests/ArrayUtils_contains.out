>>> Test 00 :contains <<<
true
--------------------------------
